index.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+// Fetch input from form
+let name = document.getElementById("name");
+let email = document.getElementById("email");
+let phone = document.getElementById("phone");
+let website = document.getElementById("website");
+let summary = document.getElementById("summary");
+let sendButton = document.getElementById("send");
```

Comment:
I prefer using `const` for variables that I don't expect to be reassigned.

Reply:
True Anson! Thank you.